style.css

```
@@ -0,0 +1,71 @@
+body {
+  background-color: rgb(141, 107, 56);
+}
+
+.classbody {
+  background-color: rgb(211, 162, 82);
+  width: 432px;
+  margin-left: 500px;
+}
+
+h2 {
+  font-family: "Anton", sans-serif;
+  margin-left: 50px;
+}
+
+p {
+  font-family: "Noto Sans JP", sans-serif;
+  text-transform: uppercase;
+  margin-left: 50px;
+}
+
+.small-logo {
+  float: right;
+  margin-right: 20px;
+}
+
+a {
+  font-family: "Noto Sans JP", sans-serif;
+  text-transform: uppercase;
+}
+
+.hashtag,
+.buttons {
+  text-decoration: none;
+}
+
+.hashtag {
+  margin-left: 50px;
+  padding-top: 20px;
+  display: inline-block;
+  color: black;
+}
+
+.buttons {
+  background-color: rgb(211, 162, 82);
+  border: 1px solid white;
+  display: inline-block;
+  width: 43%;
+  padding: 10px;
+  color: white;
+  text-align: center;
+  transition: all 300ms;
+}
+
+.buttons:hover,
+:focus {
```

Your focus pseudoclass is missing the "buttons" class name here: 

```suggestion
.buttons:focus {
```